### PR TITLE
8134 refactor to remove contactTypes (primary,  secondary, otherPetitioner) part 1

### PR DESCRIPTION
**Pull request by rosiehsmith**
- replace  contactTypes (primary,  secondary, otherPetitioner) when case not new

### CODE REVIEW

File: shared/src/business/entities/DocketEntry.js [hunk 1]
```diff
@@ -258,8 +258,13 @@ DocketEntry.prototype.generateFiledBy = function (petitioners) {
         practitioner.partyPrivatePractitioner &&
           partiesArray.push(`Counsel ${practitioner.name}`);
       });
-    const petitionersArray = this.filers.map(
-      contactId => petitioners.find(p => p.contactId === contactId).name,
+
+    const petitionersArray = this.filers.map(contactId =>
```
Comment (kkoskelin) [inline shared/src/business/entities/DocketEntry.js]: was wondering whether this does what we want, and if there's proper coverage.

File: shared/src/business/useCases/externalDocument/fileExternalDocumentInteractor.js [hunk 2]
```diff
@@ -109,6 +109,8 @@ exports.fileExternalDocumentInteractor = async (
 
   for (let [docketEntryId, metadata, relationship] of documentsToAdd) {
     if (docketEntryId && metadata) {
+      console.log('------petitioners on case', caseEntity.petitioners);
+      console.log('00000000filers', metadata);
```
Comment (matthopson) [inline shared/src/business/useCases/externalDocument/fileExternalDocumentInteractor.js]: Do we need these?

Comment (rosiehsmith) [reply shared/src/business/useCases/externalDocument/fileExternalDocumentInteractor.js]: definitely not! woops!

File: web-client/src/presenter/actions/setupContactSecondaryFormAction.test.js [hunk 3]
```diff
@@ -20,6 +21,7 @@ describe('setupContactSecondaryFormAction', () => {
           docketNumber: '101-20',
           partyType: PARTY_TYPES.petitioner,
           petitioners: [
+            MOCK_CASE.petitioners[0],
```
Comment (mrinsin) [inline web-client/src/presenter/actions/setupContactSecondaryFormAction.test.js:24]: Thank you!

Comment (rosiehsmith) [reply web-client/src/presenter/actions/setupContactSecondaryFormAction.test.js:24]: no thank YOU

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**